### PR TITLE
Add initial testing for the SUSE AI containers

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Review by dcermak — COMMENTED** [review]
LGTM, but the way I see it, we're not testing it in github actions at all? So There's really no way for us to not accidentally break their testing pipeline.

File: tests/test_ai.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    # Retry 5 times with exponential backoff delay
+    @retry(
+        wait=wait_exponential(multiplier=1, min=4, max=10),
+        stop=stop_after_attempt(5),
+    )
+    def check_ollama_response():
+        resp = requests.get(f"http://localhost:{host_port}/", timeout=30)
+        resp.raise_for_status()
+        assert "Ollama is running" in resp.text
+
```
Comment (grisu48) [inline tests/test_ai.py:36]: For now OK, I suggest to add another test that performs a query against ollama and not only check if the default page works.

**Review by grisu48 — APPROVED** [review]
Improvement suggestion for the future, for now nothing to be done.

LGTM.

**Review by alexandrevicenzi — COMMENTED** [review]
LGTM

**Pull request merged**